shared/test/otSdkDriver_test.js

```diff
@@ -159,6 +161,58 @@ describe("loop.OTSdkDriver", function() {
     });
   });
 
+  describe("#initPublisherComplete", function() {
```

Comment:
Ok, I missed this before. Can you merge this `describe("#initPublisherComplete")` with the previous one `describe("setupStreamElements")`.

Bascially you want to end up with one describe for setupStreamElements, and move the beforeEach to the top of the describe.